src/components/chips/js/chipsController.js

```diff
@@ -167,11 +166,17 @@ MdChipsCtrl.prototype.updateChipContents = function(chipIndex, chipContents){
  * Returns true if a chip is currently being edited. False otherwise.
  * @return {boolean}
  */
-MdChipsCtrl.prototype.isEditingChip = function(){
+MdChipsCtrl.prototype.isEditingChip = function() {
   return !!this.$element[0].getElementsByClassName('_md-chip-editing').length;
 };
 
 
+MdChipsCtrl.prototype.isRemovable = function() {
+  return this.readonly ?
```

Comment:
Although very succinct (:smile:), this is a bit difficult to read with a double-ternary operator. Can we refactor this into a few `if`s?